src/CMakeLists.txt

```diff
@@ -194,3 +194,13 @@ else()
       DESTINATION ${CMAKE_INSTALL_DOCDIR})
   endif()
 endif()
+
+if (NOT DEFINED CMAKE_INSTALL_PYTOOLSDIR)
+  set(CMAKE_INSTALL_PYTOOLSDIR "${CMAKE_INSTALL_DATADIR}/googlebenchmark/tools" CACHE PATH "")
+endif()
+
+if (BENCHMARK_ENABLE_INSTALL AND BENCHMARK_INSTALL_TOOLS)
+  install(
+    DIRECTORY "${PROJECT_SOURCE_DIR}/tools/"
+    DESTINATION ${CMAKE_INSTALL_PYTOOLSDIR})
```

Comment:
i'm not a python package management expert, but doesn't python expect things to be in some central location?

i see options like

```
FindPython(..)
execute_process(COMMAND ${Python_EXECUTABLE} -m site --user-site
    OUTPUT_VARIABLE PYTHON_USER_SITE
    OUTPUT_STRIP_TRAILING_WHITESPACE)
message(STATUS "installing <pybind module.so> to ${PYTHON_USER_SITE}")
install(TARGETS <module> LIBRARY DESTINATION ${PYTHON_USER_SITE})
```

offered elsewhere for this sort of thing.

i don't have any strong preference either way so if this works for your needs then its fine by me.

as for the googlebenchmark prefix: i suspect this is so the Python import paths are self-describing?

Reply:
Sure. I will create a separate issue for that, if you could just assign me, that would be great.

Reply:
i'll defer to @LebedevRI for this.

Reply:
I'd defer to whoever owns the python bindings.
Other than that, i think regardless of what happens with #2017, 
the current diff seems like a reasonable improvement.